terraform/modules/aws_gundeck-push-notifications/outputs.tf

```diff
@@ -0,0 +1,22 @@
+# Output required to configured wire-server
+
```

Comment:
Can we add:

 * "arnEnv" to the output? Might not be obvious

Reply:
From what I've see, `arnEnv == var.environment`. Though, `environment` is not a required variable. So, I guess, it makes sense output this as well. Nevertheless, I feel a bit reluctant to naming it `ancEnv`, because that is actually - or at least for me - confusing. And IMHO it is indeed just the name of the environment, which we use for prefixing. 
I think, the issue here is that the variables of the chart values are called that way. A more general name for this is probably more fitting, since other technologies will be involved in the future, too.

What do you think?

Reply:
`environment` is not required because there's always a default, right? That's fine, it must be present so the default sounds sensible.

Reply:
yes. right now we follow this approach with all modules. unfortunately for AWS, everything under one account shares a naming scope. and, as you are aware of, some things are even platform global ... m)